wallet-new/src/Cardano/Wallet/API/V1/Internal/Update.hs

```diff
@@ -0,0 +1,13 @@
+module Cardano.Wallet.API.V1.Internal.Update where
+
+import           Servant
+
+import           Cardano.Wallet.API.Response (ValidJSON)
+
+type API =
+    "update"
+    :> ( "apply"
+        :> Post '[ValidJSON] NoContent
+    :<|> "postpone"
+        :> Post '[ValidJSON] NoContent
+    )
```

Comment:
Very small remark but, I am rather _annoyed_ by these endpoints trying to play it RESTful while they are completely not. I'd rather have plain:

- `api/internal/apply-update`
- `api/internal/postpone-update`

rather than the old nesting `/update/apply`. What is the `update` resource exactly :p ? It's very minor, yet I believe we could be very honest about what those endpoint are, and they aren't RESTful.
You can probably leave it as it is now, I'll make a final "cosmetic" PR with a few stuff like that we can discuss once done.

Reply:
It's a bit of a bittersweet feeling indeed -- in my [update overhaul](https://github.com/input-output-hk/cardano-sl/pull/1884) proposal I have tried to make these all RESTful but unfortunately in the current state of things these are not. We are paying the "original sin" of the old API, in a sense. These are "imperative" endpoints to begin with, so trying to mask 'em as RESTful is probably going to be possible for clunky.

Perhaps you are right, we should be explicit here (my only consolation is that this in an internal API, hehe) and do things properly for the upcoming decentralisation.

Reply:
@adinapoli-iohk Do you think it's worthwhile to just continue using V0 update API until the new stuff is ready to go?